setup.py

```diff
@@ -305,7 +306,13 @@ def main():
         category=UserWarning,
         module='distutils.dist',
     )
-    setup(**setup_params)
+    try:
+        setup(**setup_params)
+    except RuntimeError as err:
+        if os.getenv('ANSIBLE_DEBUG'):
```

Comment:
@abadger is this change fine with you?

Reply:
I think we should just raise.  setup.py is for installing the software.  That comes with different assumptions about what you'd want to see than for running the software.

Reply:
Yeah, I'd drop the try: except altogether.

Reply:
Personally, I'd keep it all. But I don't care enough to argue :)